runtime/src/main/java/jp/co/soramitsu/runtime/network/RuntimeCall.kt

```diff
@@ -0,0 +1,35 @@
+package jp.co.soramitsu.runtime.network
+
+import io.emeraldpay.polkaj.scale.ScaleCodecReader
+import java.math.BigInteger
+import jp.co.soramitsu.fearless_utils.extensions.toHexString
+import jp.co.soramitsu.fearless_utils.runtime.AccountId
+import jp.co.soramitsu.fearless_utils.scale.dataType.uint128
+import jp.co.soramitsu.fearless_utils.wsrpc.request.runtime.RuntimeRequest
+
+sealed class RuntimeCall<T> {
+
+    abstract val path: String
+    abstract val args: ByteArray
+    abstract fun parseResult(resultBytes: ByteArray): T
+
+    sealed class NominationPoolsApi<T> : RuntimeCall<T>() {
```

Comment:
Why you need `sealed class` there? Which new inner classes will be in future?

Reply:
If the NominationPools pallet developers will decide to add more calls to this pallet. But I can't be sure we are going to use it.

Reply:
I would prefer to write as simply as possible. If we need to add new code in the future, then based on future logic, it will be easier to write more understandable code. But it's up to you :)